sdk/test/opentelemetry/sdk/trace/span_test.rb

```diff
@@ -372,14 +400,39 @@
       _(span.finish).must_equal(span)
     end
 
-    it 'sets the end timestamp' do
-      span.finish
-      _(span.to_span_data.end_timestamp).wont_be_nil
+    it 'uses the provided end timestamp' do
+      ts = Time.now
```

Comment:
Can we use some weird non-sensical timestamp here?

Reply:
✅ Explicit end timestamp is honoured

Reply:
>Can we use some weird non-sensical timestamp here?

Not sure what you mean.  As in have a separate test that receives junk instead of a valid timestamp, or just a weird fixed timestamp?

Reply:
Weird fixed timestamp - really anything other than `Time.now`, so we know that we're honouring the provided timestamp.

Reply:
Updated